dll/win32/shell32/folders/CRegFolder.cpp

```diff
@@ -284,6 +276,18 @@ HRESULT CRegFolderEnum::AddItemsFromKey(HKEY hkey_root, LPCWSTR szRepPath)
     return S_OK;
 }
 
+/*
+ * These columns try to map to CFSFolder's columns because the CDesktopFolder
+ * displays CFSFolder and CRegFolder items in the same view.
+ */
+enum REGFOLDERCOLUMNINDEX
```

Comment:
These id's are not contiguous, what happens with the id's between them?

Reply:
They are not contiguous because the regfolder has to match the fsfolder because they are displayed in the same view (desktop etc.). Only `GetDetailsEx` allows decoupling of these two folders but ROS is far away from that.

When the desktop asks a regfolder for its size (or another unhandled column), `switch(iColumn)` ends up in the default case that just returns an empty string. 

There is already a comment there 
 > Return an empty string when we area asked for a column we don't support. ...